UI/window-basic-main.cpp

```diff
@@ -7399,16 +7399,17 @@ void OBSBasic::AutoRemux(QString input, bool no_show)
 	const char *format = config_get_string(
 		config, isSimpleMode ? "SimpleOutput" : "AdvOut", "RecFormat2");
 
-	/* AV1+PCM cannot be remuxed into any supported format (until FFmpeg 6.1) */
-	if (strcmp(vCodecName, "av1") == 0 &&
-	    strncmp(aCodecName, "pcm", 3) == 0)
+	bool audio_is_pcm = strncmp(aCodecName, "pcm", 3) == 0;
+	/* Older FFmpeg versions cannot remux AV1+PCM into any supported format. */
```

Comment:
I'd appreciate a little more specificity about which versions do not support this, or what the threshold version is, here.

Reply:
Threshold should be > 6.0, though they have not tagged a release yet.